SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/NewsCollectionViewCell.swift

```diff
@@ -11,11 +11,16 @@ class NewsCollectionViewCell: UICollectionViewCell {
 
     @IBOutlet weak var newsImage: UIImageView!
     @IBOutlet weak var newsDescription: UILabel!
-    
+    @IBOutlet weak var newsButton: UIButton!
     
     override func awakeFromNib() {
         super.awakeFromNib()
         newsDescription.font = UIFont.boldSystemFont(ofSize: 20)
+        newsButton.isHidden = true
     }
 
+    @IBAction func newsButtonTapped(_ sender: Any) {
```

Comment:
why are you putting a button in the cell of the collectionview.  this is not a propper way to capture de selection of a cell delete thisa and never do such a thing again.  Please refer to the documentation of uicollectionviewcell and its delegates,